src/sentry/testutils/factories.py

```diff
@@ -973,7 +973,7 @@ def create_alert_rule(
         environment=None,
         excluded_projects=None,
         date_added=None,
-        dataset=QueryDatasets.EVENTS,
+        dataset=QueryEntity.EVENTS,
```

Comment:
Would it make sense to go all the way and also rename all constants, variables and parameters of type `QueryEntity`?

Reply:
What do you mean?

Reply:
For example, `old_entity = QueryEntity.EVENTS` instead of `old_dataset = QueryEntity.EVENTS`, or in this case, `entity=QueryEntity.EVENTS`.

Reply:
Yeah I thought about the same but I don't know .. I left it as `dataset` because the variable name on the `AlertRuleSerializer` is `dataset`, and then it would be a massive change

I am actually contemplating if this PR even makes sense. Maybe it is just enough to document `QueryDatasets` saying it maps to an `Entity` in snuba

What do you think @jjbayer ?

Reply:
No real preference here, but when in doubt, I would say take the conservative route and only add documentation.

Reply:
Yeah I agree